src/openzeppelin/tests/access/test_ownable.cairo

```diff
@@ -31,6 +33,7 @@ fn STATE() -> Ownable::ContractState {
 fn setup() -> Ownable::ContractState {
     let mut state = STATE();
     InternalImpl::initializer(ref state, OWNER());
+    testing::pop_log_raw(ZERO());
```

Comment:
why is this line needed?

Reply:
`pop_log_raw` is just `pop_log` without serde.